practice-two/src/components/ShoppingCart.jsx

```diff
@@ -29,8 +29,12 @@ const ShoppingCart = ({
   removeFromCart = () => {},
   updateQuantity = () => {},
 }) => {
+  // State to manage the status of the slider bar (open/closed)
```

Comment:
code style

Reply:
I'm having some code style issues here, formatting by prettier and eslint both disallow spaces before comments.
Specifically, if you turn off the rule of eslint, the formatting will also automatically align the space.

https://github.com/Agility-Internship/Duong-The-Hien-ReactJS/assets/113186169/a77499db-c346-4410-955b-5a5e1ac69259